tools/apiview/parsers/CONTRIBUTING.md

```diff
@@ -1,1165 +1,44 @@
-# Tree token parser
+# APIView token parsers
 
 This page describes how to contribute to [APIView](../../../src//dotnet/APIView/APIViewWeb/APIViewWeb.csproj) language level parsers.
-Specifically how to create or update a language parser to produce a hierarchy of the API using a tree data tokens, instead of a flat token list.
+Specifically how to create or update a language parser to produce a hierarchy of the API using tokens with context, instead of a flat token list.
 
-## Tree style token parser benefits
+## Benefits of token schema with parent - child context
 
-- Ability to granularly identify a specific class and it’s methods or a specific API alone within a large review without rendering entire tokens.
+- Ability to granularly identify a specific node and it's sub nodes. For e.g. if a review line for a class is available then all it's child nodes contain all methods and properties within that class.
 - Faster diffing using tree shaker instead of current text based comparison.
 - Provide diffing with context of where the line that changed belong to in the tree, instead of showing the 5 lines before and after the change.
 - Provide cross language view for a granular section within an API review.
-- Support for collapsible views at each class and namespace level to make it easier to review the change. User will be able to expand/collapse an entire class or a namespace and it’s children.
```

Comment:
you are removing this because it is not yet implemented or because with the proposed changes it won't be possible

Reply:
> you are removing this because it is not yet implemented or because with the proposed changes it won't be possible

It's not added yet. I am removing it since it's not a benefit yet straight from this new token model. We can add it once the feature is added in APIView.